### PR TITLE
Update nested-select.js

### PR DESCRIPTION
**Pull request by attiq-rehman**
Changes to use nested select for nested forms.

### CODE REVIEW

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 1]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
...
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
+              parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
+          }
+          var parentSelector = $(parentSelectorId)[0];
+
+          $(parentSelector).on('select2:select', setParentValue);
+          $(parentSelector).on('select2:unselect', setParentValue);
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 10   indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 2]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
...
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
+              parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
+          }
+          var parentSelector = $(parentSelectorId)[0];
+
+          $(parentSelector).on('select2:select', setParentValue);
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 10   indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 3]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
...
-        var parentSelectorId = '#' + model + '_' + parent;
-        var parentSelector = $(parentSelectorId)[0];
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
+              parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
+          }
+          var parentSelector = $(parentSelectorId)[0];
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 10   indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 4]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
...
       if (!!parent) {
-        var parentSelectorId = '#' + model + '_' + parent;
-        var parentSelector = $(parentSelectorId)[0];
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
+              parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
+          }
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 10   indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 5]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
...
 
       if (!!parent) {
-        var parentSelectorId = '#' + model + '_' + parent;
-        var parentSelector = $(parentSelectorId)[0];
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
+              parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 10 spaces but found 14  indent<br>Infix operators must be spaced                  space-infix-ops

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 6]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
       }
 
       if (!!parent) {
-        var parentSelectorId = '#' + model + '_' + parent;
-        var parentSelector = $(parentSelectorId)[0];
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
+          if (!$(parentSelectorId).length) {
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 10   indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 7]
```diff
@@ -150,11 +150,14 @@ var initializer = function() {
       }
 
       if (!!parent) {
-        var parentSelectorId = '#' + model + '_' + parent;
-        var parentSelector = $(parentSelectorId)[0];
-
-        $(parentSelector).on('select2:select', setParentValue);
-        $(parentSelector).on('select2:unselect', setParentValue);
+          var parentSelectorId = '#' + model + '_' + parent;
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 10   indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 8]
```diff
@@ -151,6 +151,9 @@ var initializer = function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+        if (!$(parentSelectorId).length) {
+            parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 10 spaces but found 12  indent<br>Infix operators must be spaced                  space-infix-ops

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 9]
```diff
@@ -151,6 +151,9 @@ var initializer = function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+        if (!$(parentSelectorId).length) {
+          parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Infix operators must be spaced   space-infix-ops

**Comment by mf222** [comment]
@attiq-rehman in order to be merged you need to follow the style rules in particular de [conventional commits](https://github.com/platanus/activeadmin_addons/blob/master/docs/CONTRIBUTING.md#style-rules). As you can see you did a new commit for each change, so you have 4 different commits for the same change. Please try doing a `reword` and `rebase`.

**Pull request closed without merging**